--- xorg-x11-apps-7.7/xclock-1.0.7/Clock.c.orig	2015-07-14 18:13:59.628417678 +0900
+++ xorg-x11-apps-7.7/xclock-1.0.7/Clock.c	2015-07-14 22:47:25.483396692 +0900
@@ -164,7 +164,7 @@
     {XtNbrief, XtCBoolean, XtRBoolean, sizeof(Boolean),
         offset(brief), XtRImmediate, (XtPointer) FALSE},
     {XtNstrftime, XtCString, XtRString, sizeof(String),
-        offset(strftime), XtRString, ""},
+        offset(strftime), XtRString, "%x"},
     {XtNchime, XtCBoolean, XtRBoolean, sizeof(Boolean),
 	offset(chime), XtRImmediate, (XtPointer) FALSE },
     {XtNpadding, XtCMargin, XtRInt, sizeof(int),
@@ -197,7 +197,7 @@
     {XtNminorColor, XtCForeground, XtRXftColor, sizeof(XftColor),
 	offset(minor_color), XtRString, XtDefaultForeground},
     {XtNface, XtCFace, XtRXftFont, sizeof (XftFont *),
-	offset (face), XtRString, ""},
+	offset (face), XtRString, "Sans-13"},
 #endif
 };
 
@@ -851,8 +851,8 @@
     /*
      * Compute position of tail
      */
-    x = w->clock.padding + head.xOff;
-    y = w->clock.face->ascent + w->clock.padding + head.yOff;
+    x = w->clock.padding + head.xOff / 1.3;
+    y = w->clock.face->ascent;
     /*
      * Compute bounds of tail, pad a bit as the bounds aren't exact
      */
@@ -1204,8 +1204,8 @@
         w->clock.centerY = w->core.height / 2;
     }
 #ifdef XRENDER
-    w->clock.x_scale = 0.45 * w->core.width;
-    w->clock.y_scale = 0.45 * w->core.height;
+    w->clock.x_scale = 0.4 * w->core.width;
+    w->clock.y_scale = 0.4 * w->core.height;
     w->clock.x_off = 0.5 * w->core.width;
     w->clock.y_off = 0.5 * w->core.height;
     if (w->clock.pixmap)
@@ -1405,7 +1405,7 @@
 #endif
 	    }
 	}
-	if( w->clock.analog == FALSE ) {
+	if( w->clock.analog == TRUE ) {
 	    int	clear_from = w->core.width;
 	    int i, len, prev_len;
 
@@ -1521,7 +1521,9 @@
 		     sizeof(w->clock.prev_time_string)-i);
 	    w->clock.prev_time_string[sizeof(w->clock.prev_time_string)-1] = 0;
 #endif
-	} else {
+       //} else {
+       }
+       if( w->clock.analog == TRUE ) {
 			/*
 			 * The second (or minute) hand is sec (or min)
 			 * sixtieths around the clock face. The hour hand is
